<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>Marioni Lab code repositories by MarioniLab</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">Marioni Lab code repositories</h1>
      <h2 class="project-tagline"></h2>
    </section>

    <section class="main-content">
      <h1>
<a id="introduction" class="anchor" href="#introduction" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Introduction</h1>

<p>The Marioni group is based at the Cancer Research UK Cambridge Institute and the EMBL European Bioinformatics Institute. We are involved in developing novel statistical and computational methods for analyzing single-cell RNA sequencing (scRNA-seq) data. This webpage serves as an index for our code repositories, including software packages and analysis scripts for various projects.</p>

<h1>
<a id="software" class="anchor" href="#software" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Software</h1>

<ul>
<li>The <em>scran</em> package is designed for low-level analyses of scRNA-seq data. 
It provides methods for single-cell normalization (<a href="http://dx.doi.org/10.1186/s13059-016-0947-7">Lun <em>et al.</em>, 2016</a>), cell cycle phase identification (<a href="http://doi.org/10.1016/j.ymeth.2015.06.021">Scialdone <em>et al.</em>, 2015</a>) and detection of highly variable and significantly correlated genes. 
It is officially available as part of the open-source Bioconductor project (<a href="http://bioconductor.org/packages/scran">http://bioconductor.org/packages/scran</a>), but the in-house experimental version can be obtained at <a href="https://github.com/MarioniLab/scran">https://github.com/MarioniLab/scran</a>.</li>

<li>The <em>BASiCS</em> package implements an integrated Bayesian hierarchical model to analyse scRNA-seq data by jointly performing normalization, quantification of technical variability and downstream analyses (<a href="http://dx.doi.org/10.1371/journal.pcbi.1004333">Vallejos <em>et al.</em>, 2015</a>; <a href="http://dx.doi.org/10.1186/s13059-016-0930-3">Vallejos <em>et al.</em>, 2016</a>).
It can detect highly or lowly variable genes within a population of cells, as well as differentially expressed or differentially variable genes between pre-specified populations of cells. 
<em>BASiCS</em> is currently available at <a href="https://github.com/catavallejos/BASiCS">https://github.com/catavallejos/BASiCS</a> and will shortly be submitted to Bioconductor. 
</ul>

<h1>
<a id="projects" class="anchor" href="#projects" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Projects</h1>

<ul>
<li>Simulation code for the scRNA-seq normalization-by-pooling paper (<a href="http://dx.doi.org/10.1186/s13059-016-0947-7">Lun <em>et al.</em>, 2016</a>) can be obtained at <a href="https://github.com/MarioniLab/Deconvolution2016">https://github.com/MarioniLab/Deconvolution2016</a>.</li>
<li>The scRNA-seq low-level analysis workflow based on <em>scran</em> and other Bioconductor packages can be obtained at <a href="https://github.com/MarioniLab/BiocWorkflow2016">https://github.com/MarioniLab/BiocWorkflow2016</a>.</li>
<li>A tutorial about the usage of <em>BASiCS</em> for the detection of highly/lowly variable genes can be obtained at <a href="https://github.com/catavallejos/TutorialBASiCS">https://github.com/catavallejos/TutorialBASiCS</a>.</li>
</ul>

<h1>
<a id="contact" class="anchor" href="#contact" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Contact</h1>

<p>This page is maintained by various members of the Marioni Lab. 
For information about code, contact Aaron Lun (aaron dot lun at cruk dot cam dot ac dot uk) or the authors of the individual repositories. 
For information about the group, contact John Marioni (marioni at ebi dot ac dot uk).</p>

<p><img src="tsne_marioni.png" alt=""></p>

<h1>
<a id="links" class="anchor" href="#links" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Links</h1>

<ul>
<li>CRUK homepage: <a href="http://www.cruk.cam.ac.uk/research-groups/marioni-group">http://www.cruk.cam.ac.uk/research-groups/marioni-group</a>
</li>
<li>EMBL-EBI homepage: <a href="http://www.ebi.ac.uk/research/marioni">http://www.ebi.ac.uk/research/marioni</a>
</li>
<li>GitHub page: <a href="https://github.com/MarioniLab">https://github.com/MarioniLab</a>
</li>
</ul>

      <footer class="site-footer">

        <span class="site-footer-credits">This page was generated using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>
